corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -8,6 +8,8 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         toggles = hqImport("hqwebapp/js/toggles"),
         utils = hqImport("cloudcare/js/formplayer/utils/utils");
 
+    const FormatAddress = "Address";
```

Comment:
There's a file for these, mimicking the `const.py` pattern we use in python: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/cloudcare/static/cloudcare/js/formplayer/constants.js

Reply:
moved